include/dbstructure.php

```diff
@@ -1092,39 +1092,39 @@ function db_definition($charset) {
 			);
 	$database["oembed"] = array(
 			"fields" => array(
-					"url" => array("type" => "varchar(255)", "not null" => "1", "primary" => "1"),
+					"url" => array("type" => "varbinary(255)", "not null" => "1", "primary" => "1"),
```

Comment:
Maybe. But this doesn't matter since the value is stored and fetched as binary. So a multi byte url will be split in multiple bytes - and will be merged in a single character when it is fetched.